### ISSUE TITLE
DiscordSRV giving error on startup.

### ISSUE THREAD

**Issue by ZEBaker98** [DiscordSRV/DiscordSRV#583]
Each time I start my server discordsrv reports the following error

```
[10:40:16] [DiscordSRV - Initialization/WARN]: java.lang.IllegalStateException: ServiceRegisterEvent cannot be triggered asynchronously from another thread.
[10:40:16] [DiscordSRV - Initialization/WARN]: at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:511)
[10:40:16] [DiscordSRV - Initialization/WARN]: at org.bukkit.plugin.SimpleServicesManager.register(SimpleServicesManager.java:59)
[10:40:16] [DiscordSRV - Initialization/WARN]: at github.scarsz.discordsrv.objects.metrics.BStats.<init>(BStats.java:151)
[10:40:16] [DiscordSRV - Initialization/WARN]: at github.scarsz.discordsrv.DiscordSRV.init(DiscordSRV.java:646)
[10:40:16] [DiscordSRV - Initialization/WARN]: at java.lang.Thread.run(Thread.java:748)
[10:40:16] [DiscordSRV - Initialization/ERROR]: [DiscordSRV] DiscordSRV failed to load properly: ServiceRegisterEvent cannot be triggered asynchronously from another thread.. See https://bin.scarsz.me/68e7314a-a68a-44bb-9784-618c786f5f12#satxXxMl0V5UvezMWKUyBmop7kL0YIpE for more information.
```

It seems to function normally after that but I would like to know if there is a way to fix it.

**Comment by underscore11code** [comment]
Latest DiscordSRV is 1.18.2, you are on 1.18.1

**Comment by Vankka** [comment]
This was fixed in 1.18.2
